api/src/service/config.py

```
@@ -91,5 +92,14 @@ def get_admin_url(path):
     return f"{host}{path}"
 
 
+def get_makerspace_local_timezone() -> ZoneInfo:
+    zone_str = config.get("MAKERSPACE_LOCAL_TIMEZONE")
+    try:
+        zone = ZoneInfo(zone_str)
+    except Exception as e:
+        raise (f"Variable MAKERSPACE_LOCAL_TIMEZONE not set correctly in .env, failed due to {e}")
```

You should raise an exception object, not a string